README.md

```diff
@@ -137,6 +137,7 @@ Tools for interacting with the secret network and for writing Secret Contracts
 * [Secret API](https://secretapi.io/) - Web API for using Secret Network by [Chain of Secrets][chainofsecrets]
 * [SecretJS](https://www.npmjs.com/package/secretjs) - JavaScript library for interacting with the network
 * [Secret Toolkit](https://github.com/enigmampc/secret-toolkit) - Extra tools for use in Secret Contracts
+* [Smart Contract Interface](https://www.btn.group/secret_network/smart_contract_interface) - Browser interface for smart contracts
```

Comment:
You can add:
```md
by [BTN Group][btn.group]
```
to the end of this line.
That's the point of putting the contributor link at the bottom 😉

Reply:
Whoops. Resolved. Thanks @reuvenpo!